apps/meteor/client/views/invite/InvitePage.tsx

```diff
@@ -24,9 +24,29 @@ const InvitePage = (): ReactElement => {
 	const registrationForm = useSetting('Accounts_RegistrationForm');
 	const setLoginDefaultState = useSessionDispatch('loginDefaultState');
 	const userId = useUserId();
-	const homeRoute = useRoute('home');
-	const groupRoute = useRoute('group');
-	const channelRoute = useRoute('channel');
+	const router = useRouter();
```

Comment:
I suggest moving the mutations (there are two) to their own file and avoid to use the `sdk` in the mutations, use the `useEndpoint` instead